sdk/src/main/java/com/bugsnag/android/Breadcrumb.java

```diff
@@ -46,6 +46,16 @@
         this.name = name;
     }
 
+    Breadcrumb(@NonNull String name,
+               @NonNull BreadcrumbType type,
+               @NonNull Date captureDate,
+               @NonNull Map<String, String> metadata) {
```

Comment:
It would be odd to require any code which creates a breadcrumb to create a date, wouldn't it? The default behavior should continue to be that a new breadcrumb is the current date.

Reply:
It would make it easier to test the exact timestamp of breadcrumbs as we could inject whatever date we need into the constructor, and would also save an additional method. Not a big priority though.